lib/matplotlib/backends/backend_ps.py

```diff
@@ -249,16 +254,6 @@ def __init__(self, width, height, pswriter, imagedpi=72):
 
         self._character_tracker = _backend_pdf_ps.CharacterTracker()
 
-    @cbook.deprecated("3.3")
-    @property
-    def mathtext_parser(self):
-        return MathTextParser("PS")
```

Comment:
This used to be the wrong 'since' version, I guess?

Reply:
yes (https://github.com/matplotlib/matplotlib/pull/18002)